assets/package.json

```diff
@@ -1,5 +1,5 @@
 {
   "dependencies": {
-    "solc": "^0.8.14-fixed"
+    "solc": "^0.8.17"
```

Comment:
The dep has been pinned by PR https://github.com/Magickbase/godwoken_explorer/pull/1042/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R3

Reply:
This `godwoken-v1-testnet-prod` branch not merged

Reply:
The `^` should be removed too.